src/shells/bash.rs

```diff
@@ -10,7 +10,7 @@ impl Shell for Bash {
 __pazi_add_dir() {
     # TODO: should pazi keep track of this itself in its datadir?
     if [[ "${__PAZI_LAST_PWD:-}" != "${PWD}" ]]; then
-        pazi visit "${PWD}"
+        { pazi visit "${PWD}" & } 2>/dev/null; disown
```

Comment:
I ended up switching this to:

`{ pazi visit "${PWD}" & } 2>/dev/null; disown 2>/dev/null || true`

I removed the redirect from pazi jump because pazi shouldn't be printing anything there in any case. I had to make `disown` more lenient because it can error out if `pazi visit` has already exited before it runs (`disown: no current job`), and that race ended up happening in practice.